registry/remote/registry.go

```diff
@@ -104,13 +104,21 @@ func (r *Registry) Ping(ctx context.Context) error {
 
 // Repositories lists the name of repositories available in the registry.
 // See also `RepositoryListPageSize`.
+// If last argument is NOT "", starting from the specified last non-inclusively.
```

Comment:
It'd be better to say "empty" instead of "". And is it possible to make it more concise?